src/transformers/integrations.py

```diff
@@ -1144,9 +1144,12 @@ def on_log(self, args, state, control, logs: Optional[Dict[str, float]] = None,
             return
 
         if logs is not None:
-            for name, value in rewrite_logs(logs).items():
-                if isinstance(value, (int, float)):
-                    self._metadata_namespace[name].log(value, step=state.global_step)
+            if 'train_runtime' in logs:
```

Comment:
I need a little clarification how `train_runtime` key in logs tells whether it's regular or summary metric.
I could not find anything in HF docs under `train_runtime`.